modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc

```diff
@@ -28,7 +28,7 @@ To save time, here’s the commands which you can copy and use:
 ----
 apachectl -k stop
 rsync -avz /var/www/owncloud/data /mnt/owncloud
-ln -s /mnt/owncloud /var/www/owncloud/data
+ln -s /mnt/owncloud/data /var/www/owncloud/data
```

Comment:
@mmattel was correct in [his comment](https://github.com/owncloud/docs/pull/2201#issuecomment-573400716). This will symlink `/mnt/owncloud/data` inside `/var/www/owncloud/data`. It should be changed to the following:

```console
ln -sF /mnt/owncloud/data /var/www/owncloud
```

Adding the `-F` option ensures that the target directory is removed so that the new symlink can be created without error.

Reply:
Pls, disregard the above. After a lot of thought, it makes most sense to me to replace the symlink instruction with an instruction showing how to update the ownCloud config to reference to the new directory location. Can you make that change, pls?